distribution/lib/Standard/Base/0.0.0-dev/src/Error/File_Error.enso

```diff
@@ -30,6 +35,10 @@ type File_Error
     ## Indicates that the given file's type is not supported.
     Unsupported_Type (file : File)
 
+    ## Indicates that the given type cannot be serialized to the provided file
+       format.
+    Unsupported_Output_Type (format : File_Format | File) (type : Any)
```

Comment:
Not 100% sure if these two errors belong here, but I think they kind of suit.

Reply:
Agree feels sensible.